paramak/parametric_reactors/eu_demo_2015_reactor.py

```diff
@@ -0,0 +1,421 @@
+            rotation_angle=self.rotation_angle,
+            stp_filename="divertor.stp",
+            stl_filename="divertor.stl",
+        )
+
+        # Vacuum vessel
+        vac_vessel_inner = paramak.RotateSplineShape(
+            points=[
+                (574.1267649651465, 101.10991818268735),
+                (578.248235352396, 276.2517419573795),
+                (576.2157294079991, 183.80759081183805),
+                (574.6913499497014, 165.8825455403886),
+                (592.9839034492745, 356.1350119397756),
+                (618.3902277542372, 410.1718551122461),
+                (660.0565996143763, 462.707847631295),
+                (711.8855011965004, 503.98791778753997),
+                (763.7144027786246, 528.2384308674821),
+                (800.8076362638702, 535.0747131490673),
+                (957.8187204685404, 529.1614255259608),
+                (1009.6476220506645, 511.7304778365887),
+                (1061.4765236327887, 484.3593637684569),
+                (1113.3054252149127, 445.67421552338743),
+                (1160.0530619360443, 396.81085941923095),
+                (1196.1300424490914, 343.45306647581924),
+                (1221.5363667540544, 290.06656753580216),
+                (1239.3207937675284, 234.97736402844578),
+                (1251.5158294339103, 177.39960361334886),
+                (1259.8999164545482, 119.15248445488135),
+                (1262.8978627225338, 91.48061850063345),
+                (1257.7636381996099, -69.66392221833257),
+                (1248.4670705173148, -117.9739684683608),
+                (1236.2720348509329, -162.39122749437956),
+                (1219.5038608096575, -215.31064328924913),
+                (1198.1625483934886, -267.65634357273507),
+                (1171.7399711163273, -320.87421415048993),
+                (1140.7442554642726, -372.27816805066277),
+                (1102.6347690068283, -425.0006755712611),
+                (1058.9358912022924, -476.2931568156555),
+                (1022.8589106892452, -523.1641842564957),
+                (998.4688393564809, -582.3229466099988),
+                (961.8837323573345, -633.8322193152558),
+                (788.1044741113889, -641.8422854712626),
+                (746.4381022512498, -626.4054742155761),
+                (694.6092006691258, -595.170356965071),
+                (646.3371844896964, -549.174826941633),
+                (610.7683304627485, -496.43258230780384),
+                (588.9188915604805, -442.35157799696844),
+                (578.1466100551762, -383.4976698929381),
+                (577.6991331872189, -343.8423522802541),
+                (576.2157294079991, -254.96634344146696),
+                (576.2157294079991, -201.97055742152952),
+                (576.2157294079991, -148.97477140159208),
+                (576.2157294079991, -95.97898538165487),
+            ],
+            rotation_angle=self.rotation_angle,
+            # avoid overlap between VV and blanket divertor
```

Comment:
```suggestion
```

Reply:
This line was to expplain the union of blanket and divertor. If you want to get it of it it's fine with me

Reply:
ah ok ignore this suggestion then